priv/templates/phx.gen.live/core_components.ex

```diff
@@ -370,14 +370,14 @@ defmodule <%= @web_namespace %>.CoreComponents do
   def input(assigns) do
     ~H"""
     <div>
-      <.label for={@id}><%%= @label %></.label>
+      <.label for={@id} class="mb-2"><%%= @label %></.label>
```

Comment:
https://mxstbr.com/thoughts/margin
```suggestion
    <div class="flex flex-col gap-2">
      <.label for={@id}><%%= @label %></.label>
      <.label for={@id}><%%= @label %></.label>
```